ocrd/model/ocrd_exif.py

```diff
@@ -77,14 +77,23 @@ def from_filename(image_filename):
             raise Exception("Must pass 'image_filename' to OcrdExif.from_filename")
         with exiftool.ExifTool() as et:
```

Comment:
`exiftool` still being used here?

Reply:
That was 2 commits ago, check https://github.com/OCR-D/core/pull/150/commits/015832233000e89619bc3f0edc9dd9e2196aba3f

Reply:
default value should be `1`, `-1` was for testing, I'll fix that.

Reply:
Ok, spotted it!